homeassistant/components/omnilogic/__init__.py

```diff
@@ -20,7 +20,7 @@
 
 _LOGGER = logging.getLogger(__name__)
 
-PLATFORMS = [Platform.SENSOR, Platform.SWITCH]
+PLATFORMS = ["sensor", "switch", "light"]
```

Comment:
This is reverting a change that should not be reverted :)

```suggestion
PLATFORMS = [Platform.LIGHT, Platform.SENSOR, Platform.SWITCH]
```

Reply:
Thanks - reverted back.